src/odemis/driver/hamamatsurx.py

```diff
@@ -109,6 +105,8 @@ def __init__(self, name, role, parent, spectrograph=None, daemon=None, **kwargs)
             # or the camera not being turned on.
             raise model.HwError("Failed to find readout camera, check it is powered. If powered, restart the Hamamatsu PC")
 
+        super(ReadoutCamera, self).__init__(name, role, parent=parent, **kwargs)  # init HwComponent
```

Comment:
Add a comment why we call the super method here. Raise HwError before calling super method - why did we want to do that? I think it was to ensure Odemis tries again to connect to RemoteEx. Then we had the chance to restart the Hamamatsu pc, which usually ensures that the frame grabber is working correctly again.